src/main/java/men/groupiron/GroupIronmenTrackerConfig.java

```diff
@@ -32,6 +32,26 @@ default String groupName() {
         return "";
     }
 
+    @ConfigItem(
+            keyName = "locationOption",
+            name = "Transmit In-Game Location",
+            description = "Display your player location on a map via the website. ",
+            section = groupSection
+    )
+    default boolean locationOption() {
+        return true;
+    }
+
+    @ConfigItem(
```

Comment:
Not sure if this makes sense to have since all the plugin does is send data so a user could disable it in runelite to get the same effect.

Reply:
The in-game activity one makes sense for this, as disabling the in-game activity option is essentially disabling the plugin without disabling it. However, the disable location, just prevents your team from seeing your location but allows for everything else to appear.